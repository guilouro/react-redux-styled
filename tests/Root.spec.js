import React from 'react';
import { shallow } from 'enzyme';
import Root from '../src/root';

describe('<Root />', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<Root />);
        expect(wrapper).toMatchSnapshot();
    });
});
