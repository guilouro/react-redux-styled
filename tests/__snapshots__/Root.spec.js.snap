// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> Should render correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Favicon
    alertCount={null}
    animated={true}
    animationDelay={500}
    keepIconLink={[Function]}
    url={Object {}}
  />
  <Routes />
</Provider>
`;
